### DISCUSSION TITLE
Benefits when using `null` for the whole `select` statement?

### DISCUSSION THREAD

**Discussion by bennettdams** [prisma/prisma#18075] [Q&A]
Is there a benefit of using `null` for the `select` statement if you don't need the return value?

Imagine updating a user like so:

```ts
await ctx.prisma.user.update({
  where: { id: userId },
  data: { username },
})
```
This will by default return the user object without the "nested" fields.
Imagine you don't need the user object after updating, would you have any benefits (like performance) by adding `select: null`? Like so:
```ts
await ctx.prisma.user.update({
  where: { id: userId },
  data: { username },
  select: null
})
```

**Comment by ludralph** [comment]
Hi @bennettdams 👋 

Using null will allow for some performance improvement since queries are not being made, especially for large datasets but for small datasets, the performance gain is not significant.

**Reply by bennettdams** [reply]
Thanks! Are there any evaluations for real numbers of how much improvement this would be?
And if so, maybe it would be worth the have a config (maybe globally at the Prisma client config) to use `select: null` as a default?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
